next.config.js

```diff
@@ -2,6 +2,11 @@ const path = require('path');
 const glob = require('glob');
 
 module.exports = {
+  publicRuntimeConfig: { // Will be available on both server and client
+    ALGOLIA_APP_ID: '0BJH6ZHANS',
+    ALGOLIA_API_KEY: '6b603f518f8226fe184df623bba4dce1',
```

Comment:
Is it safe to have this API key in the open?

Reply:
Yes, there's no way around this - But you can't do anything with these except query data anyway.